apps/system/test/unit/stack_manager_test.js

```diff
@@ -240,6 +240,10 @@ suite('system/StackManager >', function() {
     });
   });
 
+  test('_didntMove by default is true', function() {
+    assert.isTrue(StackManager._didntMove);
```

Comment:
this is testing internal stuff, we should do our assert on something external: either the clearTransitionClasses being called or the sheets-gesture-end event not being dispatched